test/controllers/ApplicationCompleteControllerSpec.scala

```diff
@@ -25,43 +25,128 @@ import base.SpecBase
 import generators.Generators
 import models.ValuationMethod
 import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
-import pages.ValuationMethodPage
+import pages._
 import viewmodels.checkAnswers.summary.ApplicationSummary
 import views.html.ApplicationCompleteView
 
 class ApplicationCompleteControllerSpec extends SpecBase with Generators {
 
   "ApplicationComplete Controller" - {
+    "when an Individual completes an application" - {
+      "must return OK and the correct view for a GET" in {
+        ScalaCheckPropertyChecks.forAll(arbitraryUserData.arbitrary) {
+          ua =>
+            val userAnswers       = ua.set(ValuationMethodPage, ValuationMethod.Method2).success.value
+            val Email             = "testEmail@mail.com"
+            val applicationNumber = userAnswers.applicationNumber
 
-    "must return OK and the correct view for a GET" in {
-      ScalaCheckPropertyChecks.forAll(arbitraryUserData.arbitrary) {
-        ua =>
-          val userAnswers       = ua.set(ValuationMethodPage, ValuationMethod.Method2).success.value
-          val Email             = "testEmail@mail.com"
-          val applicationNumber = userAnswers.applicationNumber
-
-          val emailUpdate   = (__ \ "applicationContactDetails" \ "email").json.put(JsString(Email))
-          val dataWithEmail = userAnswers.data.transform(__.json.update(emailUpdate)).get
-          val answers       = userAnswers.copy(data = dataWithEmail)
-          val application   = applicationBuilder(userAnswers = Option(answers)).build()
-
-          running(application) {
-            val request       =
-              FakeRequest(
-                GET,
-                routes.ApplicationCompleteController.onPageLoad(applicationNumber).url
-              )
-            implicit val msgs = messages(application)
-            val result        = route(application, request).value
-
-            val view    = application.injector.instanceOf[ApplicationCompleteView]
-            val summary = ApplicationSummary(userAnswers, AffinityGroup.Individual).removeActions
-            status(result) mustEqual OK
-            contentAsString(result) mustEqual view(applicationNumber, Email, summary)(
-              request,
-              messages(application)
-            ).toString
-          }
+            val emailUpdate   = (__ \ "applicationContactDetails" \ "email").json.put(JsString(Email))
```

Comment:
Very minor, I think `"applicationContactDetails"` can be replaced with `ApplicationContactDetailsPage.toString` if you wish to avoid having the value hardcoded.